src/pages/projectView/projectActivityPanel/screens/qr/components/QRCodeImage.tsx

```diff
@@ -18,6 +19,7 @@ import { useDebounce } from '../../../../../../hooks'
 import { lightModeColors } from '../../../../../../styles'
 import { copyTextToClipboard } from '../../../../../../utils'
 import { PaymentMethods } from '../QRCodeSection'
+import WarningIcon from './warning.svg'
```

Comment:
For SVGs, Chakra has a sweet way of converting them into Chakra Icons, this allows for automatic theming and more. Please check examples inside, src/components/icons/svg/*

In this case, I realized it is more of an image rather than an icon, so it's okay. But either way it's best to add this one, to the assets folder. We aren't keeping static images inside where the codes are so far.